calculateMetrics/metrics.py

```diff
@@ -0,0 +1,452 @@
+"""Determine how the well the team worked together"""
+
+import sys
+import os
+import numpy as np
+
+# github_data is a list containing sample data for our demo
+# github_data uses the pattern ["username", commit_total, lines_added,
+# lines_deleted, issues_opened]
+github_data = [
+    ["cassidyt2", 28, 355, 76, 2],
+    ["johnSmith", 22, 349, 50, 4],
+    ["janeDoe", 23, 375, 30, 3],
+    ["Edgar_AllenPoe", 27, 315, 75, 7],
+    ["George_Washington", 25, 360, 65, 10],
+    ["Alexander_Hamilton", 41, 530, 100, 230],
+    ["Karl_Marx", 0, 0, 0, 400],
+    ["Julius_Caesar", 25, 310, 68, 1],
+    ["Napoleon_Bonaparte", 24, 363, 70, 3],
+    ["Alexander_the_Great", 42, 540, 110, 1],
+]
```

Comment:
When will this code be modified to use data that has been retrieved from GitHub rather than fake data? I believe that this will cause some issues, but is essential to merging this PR.